robot/robot.py

```diff
@@ -32,7 +32,9 @@ def disabledInit(self): pass
     def teleopInit(self): pass
 
     def teleopPeriodic(self):
-        self.drive.move(-self.joystick.getY(), self.joystick.getX())
+        #self.drive.move(-self.joystick.getY(),self.joystick.getX())
+        #above is for normal joysticks, below is for weird black joystick with 0,0 in bottom left.
```

Comment:
Comments should start with `# ` followed by a capital letter. This doesn't apply for commented-out lines of actual code, such as 35, which should just start with the hashtag and optionally a space.